assets/scss/sidebar.scss

```diff
@@ -0,0 +1,19 @@
+.b-sidebar-backdrop.bg-dark {
+  background-color: rgb(0, 0, 0) !important;
```

Comment:
To avoid the !important, it is also possible to add our own backdrop variant, for example

```
<b-sidebar
        id="sidebar"
        bg-variant="white"
        no-header
        backdrop
        backdrop-variant="black"
        aria-label="Side navigation"
      >
```
The css could then change to:

```
b-sidebar-backdrop.bg-black {
  background-color: rgb(0, 0, 0);
  ...
```